services/user.service.js

```diff
@@ -0,0 +1,29 @@
+const UserModel = require('../models/user.schema');
+
+const getProfile = async (profileId) => {
+    return await UserModel.findOne({ profileId });
+}
+
+const getAllProfiles = async () => {
+    return await UserModel.find({});
```

Comment:
🚫 **[eslint]** <[no-return-await](https://eslint.org/docs/rules/no-return-await)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Redundant use of `await` on a return value.